resolve/workspace/LinAlgWorkspaceHIP.hpp

```diff
@@ -57,11 +60,13 @@ namespace ReSolve
 
       //info - but we need info
       rocsparse_mat_info info_A_;
-      
+
       real_type* d_r_{nullptr}; // needed for inf-norm
       real_type* norm_buffer_{nullptr}; // needed for inf-norm
+      real_type* transpose_workspace_{nullptr}; // needed for transpose
+      bool transpose_workspace_ready_{false}; // to track if allocated
```

Comment:
This is a bug. The workspace pointer must be of type `void*`.
```suggestion
      void* transpose_workspace_{nullptr}; // needed for transpose
      bool transpose_workspace_ready_{false}; // to track if allocated
```